README.md

```diff
@@ -18,6 +18,57 @@ From your [eZ Publish Platform](https://doc.ez.no/display/EZP/Installing+eZ+Publ
 php -d memory_limit=-1 composer.phar require --prefer-dist ezsystems/ez-matrix-bundle:dev-master
 ```
 
+### How to update content programmatically
+
+Here's an example on how to update the value of a matrix field for a content item. The field has two columns, and we are creating two rows of content:
+
+```
+$repository = $this->getContainer()->get( 'ezpublish.api.repository' );
+$contentService = $repository->getContentService();
+
+$repository->setCurrentUser( $repository->getUserService()->loadUser( 14 ) );
+
+$contentId = 26926;
+$newTitle = 'My updated title 2';
+
+try
+{
+    // create a content draft from the current published version
+    $contentInfo = $contentService->loadContentInfo( $contentId );
+    $contentDraft = $contentService->createContentDraft( $contentInfo );
+
+    // instantiate a content update struct and set the new fields
+    $contentUpdateStruct = $contentService->newContentUpdateStruct();
+    $contentUpdateStruct->initialLanguageCode = 'eng-US'; // set language for new version
+    $matrixValue = new \EzSystems\MatrixBundle\FieldType\Matrix\Value( array(
+                                                                              new \EzSystems\MatrixBundle\FieldType\Matrix\Row( array( 'col1' => 'row1col1', 'col2' => 'row1col2' ) ),
```

Comment:
Is there a reason why this is indented this way ? It could probably be made easier to read, as it is the main point of this piece of documentation.